submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -90,6 +90,7 @@ open class SubmissionSubmitter(
             version = nextVersion,
             method = method,
             title = submission.title,
+            processingStatus = ProcessingStatus.PROCESSED,
```

Comment:
What's the difference between `status` and `processingStatus`?

Reply:
is the same, we should named in the same way yes. I will change it